generator/osm_translator.hpp

```diff
@@ -324,6 +324,15 @@ class OsmToFeatureTranslator
     return true;
   }
 
+  bool IsCityBoundary(FeatureParams const & params)
+  {
+    feature::TypesHolder types;
```

Comment:
Maybe check that the feature is an area? Or call this from inside `IsDrawableLike(AREA)`, like Max suggests.

Reply:
But this method is only called from EmitArea, moreover, in the next PR I'll transform it to:
```c++
IsCityOrTown(Types const & types);
```